src/app/api/core/services/policies.service.ts

```diff
@@ -14,7 +14,7 @@ export class PoliciesService extends BaseService {
       [Resource.TaskTemplates]: [],
       [Resource.WorkflowState]: [UserAction.Read],
       [Resource.ViewSetting]: [],
-      [Resource.Users]: [],
+      [Resource.Users]: [UserAction.Read],
```

Comment:
This would make client users be able to fetch a complete list of users from our API which would be a security concern. It is recommended to skip the policy gate and go with a custom authorization code for this if necessary

I believe @arpandhakal has done a task similar to this before

Reply:
Fixed in commit below